### PR TITLE
vjsstandard - Tests - Tech folder 

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
run `vjsstandard` on all of the tests in the tech folder of test/unit

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Comment by misteroneill** [comment]
LGTM

File: test/unit/tech/flash.test.js [hunk 1]
```diff
@@ -1,116 +1,131 @@
...
+// fake out the <object> interaction but leave all the other logic intact
+class MockFlash extends Flash {
+  constructor() {
+    super({});
+  }
+}
 
-test('Flash.canPlaySource', function() {
-  var canPlaySource = Flash.canPlaySource;
+QUnit.module('Flash');
```
Comment (gkatsev) [inline test/unit/tech/flash.test.js]: new line here.

File: test/unit/tech/html5.test.js [hunk 2]
```diff
@@ -1,381 +1,451 @@
...
 import document from 'global/document';
 
-q.module('HTML5', {
-  'setup': function() {
+QUnit.module('HTML5', {
+  setup() {
 
-    el = document.createElement('div');
-    el.innerHTML = '<div />';
+    this.el = document.createElement('div');
```
Comment (gkatsev) [inline test/unit/tech/html5.test.js]: why change this one and not `player` and `tech` as well?

File: test/unit/tech/html5.test.js [hunk 3]
```diff
@@ -1,381 +1,451 @@
...
+    this.el.innerHTML = '<div />';
 
     player = {
-      id: function(){ return 'id'; },
-      el: function(){ return el; },
+      id() {
+        return 'id';
+      },
+      el: () => {
+        return this.el;
```
Comment (gkatsev) [inline test/unit/tech/html5.test.js]: I'm not sure this is right anymore with the `this.el` change.

File: test/unit/tech/tech-faker.js [hunk 4]
```diff
@@ -1,24 +1,23 @@
...
   }
 
   createEl() {
-    var el = super.createEl('div', {
+    const el = super.createEl('div', {
       className: 'vjs-tech'
     });
 
-    /*if (this.player().poster()) {
+    /* if (this.player().poster()) {
```
Comment (gkatsev) [inline test/unit/tech/tech-faker.js]: just delete this comment.

File: test/unit/tech/html5.test.js (hunk 2 above)
Comment (BrandonOCasey) [inline test/unit/tech/html5.test.js]: el was being redefined in every test and eslint was complaining about it

Comment (gkatsev) [inline test/unit/tech/html5.test.js]: and it didn't complain about `player` and `tech`? Though, I think it breaks `player.el()` below.

Comment (BrandonOCasey) [inline test/unit/tech/html5.test.js]: should I just change everything to `this.el` I think that would be a cleaner change than worrying about shared globals like it is right now

Comment (gkatsev) [inline test/unit/tech/html5.test.js]: Actually, I'm not sure that this `el` is used anywhere *except* on line 20, so, we could probably just make it a local variable?

Comment (BrandonOCasey) [inline test/unit/tech/html5.test.js]: yeah that works

**Comment by BrandonOCasey** [comment]
@gkatsev made `el` a local `const` in the `setup()` function. I think that should take care of the last issue with this PR

**Comment by gkatsev** [comment]
LGTM

**Pull request merged**